packages/nextra/src/loader.ts

```diff
@@ -159,34 +176,50 @@ async function loader(
     newNextLinkBehavior
   }
 
+  const pageNextRoute =
+    '/' +
+    path
+      .relative(pagesDir, resourcePath)
+      // Remove the `mdx?` extension
+      .replace(MARKDOWN_EXTENSION_REGEX, '')
+      // Remove the `*/index` suffix
+      .replace(/\/index$/, '')
+      // Remove the only `index` route
+      .replace(/^index$/, '')
+
   return `
-import { withSSG as __nextra_withSSG__ } from 'nextra/ssg'
+import { SSGContext as __nextra_SSGContext__ } from 'nextra/ssg'
+
 import __nextra_withLayout__ from '${layout}'
+${themeIncludeStyles && `import '${layout}/style.css'`}
```

Comment:
Same here

Reply:
Also updated this to keep it the same format:

```
${layoutConfig ? `import __nextra_layoutConfig__ from '${layoutConfig}'` : ''}
```